bundles/configuration.rst

```diff
@@ -319,12 +319,16 @@ In your extension, you can load this and dynamically set its arguments::
             // ... now use the flat $config array
         }
 
-Using the Bundle Class
-----------------------
+Using the AbstractBundle Class
+------------------------------
 
 .. versionadded:: 6.1
 
     The ``AbstractBundle`` class was introduced in Symfony 6.1.
+    
+.. caution::
+
+    This is an alternative implementation to the above mentioned Bundle class. Bundle and AbstractBundle are not compatible.
```

Comment:
I'd say after the first sentence something like...
> You can use one of these approaches but not both at the same time.

that would make clearer what "not compatible" means in this context.

Reply:
Thx, I'm definitely not a native speaker :)